analyzers/its/sources/Net5/Net5/S3776.cs

```diff
@@ -2,6 +2,22 @@
 {
     public class S3776
     {
+        string field;
+        private int one, two;
+        string Property
```

Comment:
_Nitpick:_ Missing EOL
```suggestion
        private int one, two;
        string Property
```